### ISSUE TITLE
[Suggestion] Late Game Mechanical Power

### ISSUE THREAD

**Issue by Blanclour** [StrangeLoopGames/EcoSuggestions#773]
The late game industry skill recipes require a lot of the mechanics skill recipes. Problem is that the mechanics skill tables are powered by mechanical energy while industry tables are powered by electricity.

Yes you could make windmills and waterwheels and place them on your fancy high tech factory to power these older machines, but that would not only kill the modern aesthetic but is not something you would find in modern factories in most cases.

I suggest an electric or gas powered motor object that produces mechanical energy in the late game. This would make mechanical energy easier to produce in compact spaces, and would fit nicely with the modern aesthetic.

**Comment by ThePiachu** [comment]
Yeah, it would be nice not to make my house look like some wooden cog punk ;)

![image](https://user-images.githubusercontent.com/964559/79674548-03166480-8199-11ea-822a-9077317367b6.png)

Trying to run two stamp mills, two iron shakers, a sawmill and a machinist setup takes so many of those mechanical power producers!
